src/Jackett.Common/Definitions/pornolab.yml

```
@@ -0,0 +1,253 @@
+    - {id: 1787, cat: XXX, desc: "Азиатские гей-фильмы в высоком качестве (DVD и HD) / High-Quality Full Length Asian Movies (Gay DVD & HD)"}
+    - {id: 1763, cat: XXX/Pack, desc: "ПАКи гей-роликов и сайтрипов / Clip's & SiteRip's Packs (Gay)"}
+    - {id: 1777, cat: XXX, desc: "Гей-ролики в высоком качестве (HD Video) / Gay Clips (HD Video)"}
+    - {id: 1691, cat: XXX, desc: "Ролики, SiteRip'ы и сцены из гей-фильмов / Clips & Movie Scenes (Gay)"}
+    - {id: 1692, cat: XXX/ImageSet, desc: "Гей-журналы, фото, разное / Magazines, Photo, Rest (Gay)"}
+    - {id: 1720, cat: XXX, desc: "Архив (Гей-порно)"}
+
+  modes:
+    search: [q]
+
+settings:
+  - name: username
+    type: text
+    label: Username
+  - name: password
+    type: password
+    label: Password
+  - name: stripcyrillic
+    type: checkbox
+    label: Strip Cyrillic Letters
+    default: false
+  - name: sort
+    type: select
+    label: Sort requested from site
+    default: 1
+    options:
+      1: created
+      10: seeders
+      7: size
+      2: title
+  - name: type
+    type: select
+    label: Order requested from site
+    default: 2
+    options:
+      2: desc
+      1: asc
+
+login:
+  path: forum/login.php
+  method: form
+  form: form[action="login.php"]
+  captcha:
+    type: image
+    selector: img[src*="/captcha/"]
+    input: input[name^="cap_code_"]
+  inputs:
+    input[name="login_username"]: "{{ .Config.username }}"
+    input[name="login_password"]: "{{ .Config.password }}"
+  selectors: true
+  error:
+    - selector: div:contains("Форум временно отключен")
+    - selector: h4.warnColor1.tCenter.mrg_16
+  test:
+    path: forum/index.php
+    selector: div.topmenu a[onclick*=logout]
+
+search:
+  paths:
+    - path: forum/tracker.php
+  inputs:
+    $raw: "{{ if .Categories }}{{ range .Categories }}f[]={{.}}&{{end}}{{ else }}f[]=-1{{ end }}"
+    o: "{{ .Config.sort }}"
+    s: "{{ .Config.type }}"
+    nm: "{{ .Keywords }}"
+
+  keywordsfilters:
+    - name: re_replace
+      args: ["\\W+", " "]
+    - name: trim
+
+  rows:
+    selector: table#tor-tbl > tbody > tr:has(a.tr-dl)
+
+  fields:
+    category:
+      selector: a.f
+      attribute: href
+      filters:
+        - name: querystring
+          args: f
+    title:
+      selector: a.tLink
+      filters:
+        - name: re_replace
+          args: ["(\\([\\p{IsCyrillic}\\W]+\\))|(^[\\p{IsCyrillic}\\W\\d]+\\/ )|([\\p{IsCyrillic} \\-]+,+)|([\\p{IsCyrillic}]+)", "{{ if .Config.stripcyrillic }}{{ else }}$1$2$3$4{{ end }}"]
+        - name: trim
+    details:
+      selector: a.tLink
+      attribute: href
+    download:
+      selector: a.tr-dl
+      attribute: href
+    size:
+      selector: td:nth-child(6) u
+    date:
+      # unix
+      selector: td:nth-child(11) u
+    seeders:
+      selector: td.seedmed > b
+    leechers:
+      selector: td.leechmed > b
+    grabs:
+      selector: td:nth-child(9)
+    downloadvolumefactor:
+      text: 0
+    uploadvolumefactor:
+      text: 1
```

```suggestion
    downloadvolumefactor:
      text: 1
    uploadvolumefactor:
      text: 1
    minimumratio:
      text: 0.31
```

C# indexer has dlvf as `1`, and https://github.com/Jackett/Jackett/issues/9456 states there's no freeleech, which seems to be the case from a quick browse.

[Rule 4.5](https://pornolab.net/forum/viewtopic.php?t=980423) also says there's a global minimum ratio of `>0.3`, so we could add that as well.